packages/ddex/ingester/parser/ern38x.go

```diff
@@ -163,13 +163,36 @@ func parseERN38x(doc *xmlquery.Node, crawledBucket, releaseID string) (tracks []
 		}
 	}
 
+	// Parse <ReleaseDeal>s from <DealList>
+	dealNodes := xmlquery.Find(doc, "//DealList/ReleaseDeal")
+	if len(dealNodes) == 0 {
+		errs = append(errs, fmt.Errorf("no <ReleaseDeal> found"))
+		return
+	}
+
+	refToTrackReleaseMap := make(map[string]*common.CreateTrackRelease)
+	refToAlbumReleaseMap := make(map[string]*common.CreateAlbumRelease)
+	for i := range tracks {
+		refToTrackReleaseMap[tracks[i].DDEXReleaseRef] = &tracks[i]
+	}
+	for i := range albums {
+		refToAlbumReleaseMap[albums[i].DDEXReleaseRef] = &albums[i]
+	}
+	for _, dNode := range dealNodes {
+		err := processDealNode(dNode, refToTrackReleaseMap, refToAlbumReleaseMap)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+	}
+
 	return
 }
 
 // processReleaseNode parses a <Release> into a CreateTrackRelease or CreateAlbumRelease struct.
 func processReleaseNode(rNode *xmlquery.Node, soundRecordings *[]SoundRecording, images *[]Image, crawledBucket, releaseID string) (track *common.CreateTrackRelease, album *common.CreateAlbumRelease, err error) {
 	releaseRef := safeInnerText(rNode.SelectElement("ReleaseReference"))
-	releaseDateStr := safeInnerText(rNode.SelectElement("GlobalOriginalReleaseDate")) // TODO: This is deprecated. Need to use DealList
+	releaseDateStr := safeInnerText(rNode.SelectElement("GlobalOriginalReleaseDate"))
```

Comment:
Note: we don't require a deal for the album (see the CPD examples). If there's no album deal, it falls back to using this deprecated field.